### ISSUE TITLE
Subjects: Links not distinguishable without relying on color

### ISSUE THREAD

**Issue by RyanAJensen** [pulibrary/DSS#576]
### Steps to recreate
Visit /subjects/_<subject.id>_/edit and run the Axe dev tools

### URL
http://&#8203;localhost:3000/subjects/_<subject.id>_/edit

### Issues
Links not distinguishable without relying on color

### Severity
High

### WCAG Success Criterion
- [ ] Links must be distinguishable without relying on color (2)

### WCAG Conformance level (A, AA, AAA)
AA

### User impact
Not everyone perceives colors the same way, with some users having difficulty due to partial sight, color blindness, or monochrome displays. To improve accessibility, information should be conveyed through text cues or other visual indicators, not just color.

### Recommended fix
To solve this problem, you need to fix at least (1) of the following:
- The link has insufficient color contrast of 2.53:1 with the surrounding text. (Minimum contrast is 3:1, link text: #2762ae, surrounding text: #212529)
- The link has no styling (such as underline) to distinguish it from the surrounding text

### Acceptance criteria
- [ ] I can successfully use this feature with Voice Over in Safari.
- [ ] I have checked this feature using one of the following browser extensions and confirmed that it does not add any level A or level AA issues:
    * axe dev tools
    * Lighthouse
    * Wave
- [ ] WCAG Success Criterion are resolved

### Notes
2 instances with the show and back buttons

**Comment by RyanAJensen** [comment]
Solved accidentally with code included in an unrelated PR (I have learned a lot since then!)

**Comment by kevinreiss** [comment]
Ok @RyanAJensen. I will close out this one.
